src/olympia/versions/models.py

```diff
@@ -675,6 +675,22 @@ def transformer_license(cls, versions):
             if license:
                 version.license = license
 
+    def transformer_auto_approvable(cls, versions):
+        """Attach  auto-approvability information to the versions."""
+        ids = set(v.id for v in versions)
+        if not ids:
+            return
+
+        auto_approvable = dict(
+            Version.objects.auto_approvable()
+                   .annotate(
+                       is_ready_for_auto_approval=Value(True, BooleanField()))
+                   .values_list('pk', 'is_ready_for_auto_approval')
+        )
```

Comment:
So, this is just a dict of `pk: True`?  Why not go with the more simple solution of getting a list of pks and setting true if the version pk is in that list?

Reply:
Yeah, also, I should filter by `pk__in=ids` ... I'll change this.

Reply:
https://github.com/mozilla/addons-server/pull/16105/commits/5263dc2c5789318325bb4b5fce314a09bc21a3bd